#150798546 Fix tests for accounts

rosemaina opened the pull request:
#### What does this PR do?
Fixed and added appropriate account tests
 
#### Description of Task to be completed?
Added and refactored some tests in the files below:
-test_login.py
-test_check_token.py
-test_profile.py
-test_switch_team.py
-test_team_access_middleware.py

#### How should this be manually tested?
After cloning the repo, cd into it and run this command `./manage.py test hc/accounts`

#### What are the relevant pivotal tracker stories?
#150798546

Commit:
bg(tests): Add tests for login

- Added tests to check email content and check if user was created
- Fixed test.py to give team access to Bob

lornatumuhairwe committed:
ch(tests): Remove team access properties from Bob and Charlie

lornatumuhairwe committed:
ch(tests): Remove team access properties from Bob and Charlie

ch(integrations): Updated README.MD file

  - Added Travis and Coveralls badges to README.md
  - Update the git clone information to match our repository
  - Added coverage and mock to the requirements for coverage and testing respctively
  - Added .idea to .gitignore to cater for developers using IDEs

[Finishes #150761656]

ch(intgrations):integrate slack with travis ci

	-Add slack token to the travis.yml file

[Finishes #150780738]

bg(accounts tests): Added and refactored some tests in the files below:
-test_login.py
-test_check_token.py
-test_profile.py
-test_switch_team.py
-test_team_access_middleware.py

[Finishes #150798546]

bg(accounts tests): Added and refactored some tests in the files below:
-test_login.py
-test_check_token.py
-test_profile.py
-test_switch_team.py
-test_team_access_middleware.py

[Finishes #150798546]

bg(tests): Add tests to test_profile.py

- Added tests for creating and revoking and API token
- Added tests for checking whether emails are sent when
inviting a team member and sending reports

ch(tests):Add test check redirect logged in

	-Checks that you are redirected when logged in

ch(urls): Fix indentation error

bg(tests): Associated new user with check

bg(tests): Tests for check token for login
-This was done using alice's token set in the setup function

bg(tests): Cleaned up tests
- removed unnecessary print statements and added comments

bg(tests): Uniformity in tests organization
- Adjusted the code to follow the PEP8 standard
- written more descriptive code. eg, replaced r with response

bg(tests): Removed non uniform docstrings and grammatic errors in commits

bg(tests):Refactor test redirects when logged in

	- Tests redirects when you are already logged in

ch(gitignore): Add htmlcov and .vscode

lornatumuhairwe committed:
bg(tests): Add tests to test_profile.py

- Added tests for creating and revoking and API token
- Added tests for checking whether emails are sent when
inviting a team member and sending reports

Commit:
ch(tests):Add test check redirect logged in

	-Checks that you are redirected when logged in

Commit:
ch(urls): Fix indentation error

lornatumuhairwe committed:
bg(tests): Tests for switch teams
- tests for checking team membership
- tests for wsitching to own team

lornatumuhairwe committed:
bg(tests): Associated new user with check

Commit:
bg(tests): Tests for check token for login
-This was done using alice's token set in the setup function

lornatumuhairwe committed:
bg(tests): Cleaned up tests
- removed unnecessary print statements and added comments

lornatumuhairwe committed:
bg(tests): Uniformity in tests organization
- Adjusted the code to follow the PEP8 standard
- written more descriptive code. eg, replaced r with response

lornatumuhairwe committed:
bg(tests): Test whether a new profile is added in association with the new user
- a new user was created and a profile associated with the user is created
- this profile is queried to ensure that it is associated with the new user

lornatumuhairwe committed:
bg(tests): Removed non uniform docstrings and grammatic errors in commits

lornatumuhairwe committed:
bg(tests): Replaced r with response to have well documented code

Commit:
bg(tests):Refactor test redirects when logged in

	- Tests redirects when you are already logged in

Commit:
ch(gitignore): Add htmlcov and .vscode

rosemaina committed:
Merge branch 'develop' into bg-accounts-tests-150798546

judithachieng commented:
@rosemaina @lornatumuhairwe please squash some commits, these are too many .

andela-jmwalo commented on hc/accounts/tests/test_check_token.py:
```diff
@@ -3,26 +3,38 @@
+        self.assertRedirects(response, "/accounts/login/")
+
 
-    ### Login with a bad token and check that it redirects
```
remove this line it's unecessary

andela-jmwalo commented on hc/accounts/tests/test_check_token.py:
```diff
@@ -3,26 +3,38 @@
 
-    ### Login with a bad token and check that it redirects
 
-    ### Any other tests?
```
remove this too, although you should have also come up with more tests

lornatumuhairwe committed:
bg(accounts tests): Added and refactored some tests in the files below:
-test_login.py
-test_check_token.py
-test_profile.py
-test_switch_team.py
-test_team_access_middleware.py

[Finishes #150798546]

lornatumuhairwe committed:
ch(
.gitignore): Resolved merge conflicts

lornatumuhairwe committed:
bg(accounts tests): Added and refactored some tests in the files below:
-test_login.py
-test_check_token.py
-test_profile.py
-test_switch_team.py
-test_team_access_middleware.py

[Finishes #150798546]

lornatumuhairwe committed:
Merge branch 'bg-accounts-tests-150798546' of https://github.com/andela/healthchecks-invincibles into bg-accounts-tests-150798546

lornatumuhairwe committed:
Merge branch 'bg-accounts-tests-150798546' of https://github.com/andela/healthchecks-invincibles into bg-accounts-tests-150798546

lornatumuhairwe committed:
bg(tests): Fixed merge err in test_login file

lornatumuhairwe committed:
Merge branch 'develop' into bg-accounts-tests-150798546
